src/vsphere_cpi/lib/cloud/vsphere/vm_type.rb

```diff
@@ -12,7 +12,8 @@ def initialize(datacenter, cloud_properties)
       @cloud_properties = cloud_properties
     end
 
-    %w[cpu ram disk cpu_hot_add_enabled memory_hot_add_enabled nsx vmx_options nsxt nested_hardware_virtualization upgrade_hw_version datacenters ].each do |name|
+    %w[cpu ram disk cpu_hot_add_enabled memory_hot_add_enabled nsx vmx_options
+       nsxt nested_hardware_virtualization upgrade_hw_version datacenters gpu vm_group ].each do |name|
```

Comment:
How did gpu come here?

Which branch have you rebased it off?

We are not supporting any gpu stuff